src/xai_grok/schemas/api_key.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+import typing
+
+import pydantic
+
+
+class APIKey(pydantic.BaseModel):
+    acls: typing.List[str]
+    api_key_blocked: bool
+    api_key_disabled: bool
+    api_key_id: str
+    create_time: datetime.datetime
+    modified_by: str
+    modify_time: typing.Union[datetime.datetime, typing.Literal[""]]
```

Comment:
The `modify_time` field allows an empty string, which can lead to runtime errors when datetime operations are performed. Consider using `Optional[datetime.datetime]` instead.